### DISCUSSION TITLE
PSRule.Rules.Azure/customization/storing-custom-rules/

### DISCUSSION THREAD

**Discussion by giscus** [Azure/PSRule.Rules.Azure#2555] [Documentation]
# PSRule.Rules.Azure/customization/storing-custom-rules/

Ready to go rules for testing Azure Infrastructure as Code (IaC).

https://azure.github.io/PSRule.Rules.Azure/customization/storing-custom-rules/

**Comment by heymikeey** [comment]
The link "Packaging rules in a module" seems to be broken

**Reply by BernieWhite** [reply]
@heymikeey Thanks for reporting.

**Reply by BernieWhite** [reply]
This has been updated.
